etc/conf_templates/cron_lab_tests.jinja2

```diff
@@ -1 +1 @@
-0/5 * * * * {{ unix_user }} {{ virtualenv }}/bin/python {{ project_dir }}/manage.py batch_load >> /usr/lib/ohc/log/cron_synch.log 2>&1
+*/15 * * * * {{ unix_user }} {{ virtualenv }}/bin/python {{ project_dir }}/manage.py batch_load >> /usr/lib/ohc/log/cron_synch.log 2>&1
```

Comment:
This isn't what I was expecting...
According to this website this is 'load every 15 mins instead of every 5'

New:
https://cronexpressiondescriptor.azurewebsites.net/?expression=*%2F15+*+*+*+*&locale=en
Old:
https://cronexpressiondescriptor.azurewebsites.net/?expression=0%2F5+*+*+*+*+&locale=en

Reply:
I think that might be true for Azure but not for traditional linux. Scary that it appears highly on cron descriptors google search.

I for one am slightly wary of our new github overlords.